tooling/components/io.siddhi.distribution.editor.core/src/main/resources/web/editor/commons/js/workspace/workspace.js

```diff
@@ -550,7 +550,7 @@ define(['ace/ace', 'jquery', 'lodash', 'log','dialogs','./service-client','welco
                 if(_.isNil(this._deleteFileDialog)){
                     this._deleteFileDialog = new Dialogs.DeleteConfirmDialog(app);
                 }
-                this._deleteFileDialog.render();
+                this._deleteFileDialog.render(options);
```

Comment:
Options should be passed when creating the dialog not during render

Reply:
This Dialog object is only created once So filename is parsed through render

Reply:
Ack